horace_core/sqw/PixelData/@PixelTmpFile/PixelTmpFile.m

```diff
@@ -1,5 +1,21 @@
 classdef PixelTmpFile < handle
-% Class to handle deletion of TmpFile on closure
+% Class temporary PixelDataFileBacked files
+%
+% Created by PixelData file-backed on starting modifying operation
```

Comment:
```suggestion
% Created by PixelData file-backed on starting modifying operation.
```
Based on the assumption that the next line starts a new sentence. If PixelDataFileBacked.get_new_handle is a qualifier for operation then that is not the case, and there should be a which before stores in L5. Basic point is that this needs separating out somehow to be understandable